src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/ExportToolbarAction.js

```diff
@@ -116,6 +116,16 @@ export default class ExportToolbarAction extends AbstractListToolbarAction {
     };
 
     @action handleConfirm = () => {
+        let filters = {};
+        if (Object.keys(this.listStore.filterQueryOption).length > 0) {
+            filters = this.listStore.filterQueryOption;
+        }
```

Comment:
Instead of using `let` we try to make use of `const` as much as possible. You can achieve that using a ternary operator. We also make use of destructuring a lot, to avoid repeating certain stuff. So I would change that section to:

```javascript
const {filterQueryOption} = this.listStore;
const filter = Object.keys(filterQueryOption).length > 0 ? filterQueryOption : undefined;
```

Reply:
Thanks for the tip @danrot! Refactored in latest commit.